src/api/Feed.js

```diff
@@ -259,7 +259,7 @@ class Feed extends Base {
         assignmentAPI.updateTaskAssignment({
             file,
             taskAssignmentId,
-            resolutionState,
+            taskStatus,
             message,
```

Comment:
Is message still used? Seems like it could be removed now, since we were only using it for Accepted/Rejected previously AFAIK.